streams/id.m3u

```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" status="online",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
-#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
-#EXTINF:-1 tvg-id="" status="online",iNews (720p)
+#EXTINF:-1 tvg-id="Indosiar.id" status="online",Indosiar (720p)
+http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
 https://vcdn2.rctiplus.id/live/eds/inews_fta/live_fta/inews_fta.m3u8
-#EXTINF:-1 tvg-id="KompasTV.id" status="error",Kompas TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/874-kompas-tv
+#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",JakartaGlobe News (540p) [Not 24/7]
+http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
+#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",JakartaGlobe News (720p) [Not 24/7]
+https://b1english.beritasatumedia.com/Beritasatu/B1English_1280x720.m3u8
+#EXTINF:-1 tvg-id="KompasTV.id" status="online",Kompas TV (720p)
+https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(dc9040ad-a4e1-4d08-883c-a65542b69602)/index.m3u8
 #EXTINF:-1 tvg-id="MetroTV.id" status="online",Metro TV (360p) [Not 24/7]
 http://edge.metrotvnews.com:1935/live-edge/smil:metro.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="MNCTV.id" status="online",MNCTV (720p)
 https://vcdn2.rctiplus.id/live/eds/mnctv_fta/live_fta/mnctv_fta.m3u8
-#EXTINF:-1 tvg-id="MyTV.id" status="error",MyTV (720p) [Not 24/7]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/6711-mytv
-#EXTINF:-1 tvg-id="NetTV.id" status="error",Net. TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/875-net-tv
-#EXTINF:-1 tvg-id="OChannel.id" status="error",O Channel [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/206-ochannel
```

Comment:
Over local verification, seems to work. I don't see why it's taken off.

Reply:
![Screenshot_20220308-203310](https://user-images.githubusercontent.com/76893443/157248100-f83539aa-779b-42f3-9534-67dcd4fa1405.png)

Reply:
Well I tried using my Windows streamlink and it works. Maybe it's due to something else.

Only @dct-infra can tell us what was going on. I know lots of Indonesians / Malaysians used my old "query-streamlink" heroku service for getting access to these streams, and it sometimes had cut off. Maybe it's due to rate limiting or throttling too.

Reply:
@LaneSh4d0w The front is currently russian but it changes randomly every other week or month.
The QS however is multi-homed and queries can be made from 2 to several EU or russian locations, whether round robin or specifically chosen, it depends on rate limiting and other usual issues.
QS logs show code 200 on several EU backends.

Reply:
I guess the Russian server down due to the Russia-Ukraine situation

Reply:
> @LaneSh4d0w The front is currently russian but it changes randomly every other week or month. The front is invisible to destination servers (YT, DLM, Twitch etc) most of the time, except when a QS is active on the current front. So QS is multi-homed and queries can be made from 2 to several EU or russian locations, whether round robin or specifically chosen, it depends on rate limiting and other usual issues. see the picture? Concerning our issue, QS logs show code 200 on several backends for the listed channels, which is not a proof of a successful query in QS world. A simple wide [Geo-Block] imho.

Most certainly this happening. I was wondering about a proxy pool for certain DM geo-blocked links.